wafer/pages/models.py

```diff
@@ -6,11 +6,12 @@
 from django.core.exceptions import ValidationError, NON_FIELD_ERRORS
 from django.conf import settings
 from django.db import models
+from django.core.cache import caches
 from django.db.models.signals import post_save
 from django.utils.encoding import python_2_unicode_compatible
 
 
-from markitup.fields import MarkupField
+from markitup.fields import MarkupField, render_func
```

Comment:
Just wanted to double-check that render_func gets all the settings and extensions from settings since we're not actually testing that and the initial rendering goes via the MarkupField base logic.